community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/ListIndexesProcedure.java

```diff
@@ -46,6 +49,7 @@ protected ListIndexesProcedure(ProcedureName procedureName)
         super( procedureSignature( procedureName )
                 .out( "description", Neo4jTypes.NTString )
                 .out( "state", Neo4jTypes.NTString )
+                .out( "unique", Neo4jTypes.NTBoolean )
```

Comment:
IMO, this will be problematic as we expand on what types of indexes we allow - we'd need a boolean for each type. In the longer term, we might want to have a `type` column here, with stuff like `exact_label_property`, `path`, `unique_label_property` and so on. But we don't have an agreed upon naming scheme for the index types. Could we get away with adding "UNIQUE" to the description of unique indexes (eg. in the `userDescription()` method)? That'd make them match cypher as well:

    UNIQUE INDEX ON :User(name)